src/style.css

```diff
@@ -0,0 +1,85 @@
+@font-face {
+  font-family: Avenir;
+  font-style: normal;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 900;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-weight: 300;
+}
+
+.Search {
+  position: relative;
+  margin-top: 20px;
+}
+
+.inputField {
+  width: 100%;
+  box-sizing: border-box;
+  font-family: "Avenir", "Arial", sans-serif;
+  font-style: normal;
+  color: #3d4e61;
+  background-color: #fff;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+}
+
+.inputBig {
+  padding-top: 23px;
+  padding-left: 62px;
+  padding-bottom: 23px;
+  font-size: 16px;
+}
+
+.inputSmall {
+  padding-top: 10px;
+  padding-left: 33px;
+  padding-bottom: 10px;
+  font-size: 14px;
+}
+
+.inputField:hover {
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.inputField:focus {
+  font-weight: 900;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border-radius: 3px;
+  outline: none;
+}
+
+.Img {
+  position: absolute;
+  display: block;
+  padding: 0;
+  border: 0;
+  background: none;
+}
+
+.imgBig {
+  top: 25px;
+  left: 26px;
+}
+
+.imgBig img {
```

Comment:
Don't use tag names in styles. Use classes instead. You can add attribute class on the image